.eslintignore

```diff
@@ -1,3 +1,6 @@
 node_modules
 coverage
-testresults
\ No newline at end of file
+testresults
```

Comment:
I might be turned around, but is this configuration reversed? when we run eslint against the generator, we do want it to hit the transpiled index.js, tests, etc. I was originally hoping we could configure things such that when we would lint the generator codebase, we could also get it to lint the template files, but I kept encountering failures when it did that because the template files have variables/substitution tokens/etc. that make the linters error out